attribute/set_test.go

```diff
@@ -225,3 +325,45 @@ func args(m reflect.Method) []reflect.Value {
 	}
 	return out
 }
+
+func BenchmarkFiltering(b *testing.B) {
+	var kvs [26]attribute.KeyValue
+	buf := [1]byte{'A' - 1}
+	for i := range kvs {
+		buf[0]++ // A, B, C ... Z
+		kvs[i] = attribute.String(string(buf[:]), "")
+	}
+
+	var result struct {
+		set     attribute.Set
+		dropped []attribute.KeyValue
+	}
+
+	run := func(fltr attribute.Filter) func(*testing.B) {
```

Comment:
Can we rename `run` to `benchFn`?

I would find  `b.Run("NoFilter", benchFn(nil))` more readable than `b.Run("NoFilter", run(nil))`